sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,906 @@
+the two resulting Stacker sets and wallets each own BTC on their respective
+forks.  This means that Stackers do not lose money from forks, nor do forks
+pose a risk to the safety of users' BTC.
+
+* We use the fact that PoX pays BTC
+to STX holders to propose a liveness recovery mechanism, whereby some BTC
+payouts are repurposed to fulfill peg-out requests should the custodian fail.
+
+* We reward Stackers for staying online to process peg operations by
+compensating them with BTC via PoX, regardless of the peg's operation volume.  Stackers
+are compensated proportional to their signing power on the peg (i.e how many
+STX they locked) regardless of how many peg transactions they process.
+
+By leveraging these properties, sBTC offers the following advantages over the
+state-of-the-art:
+
+* **If a majority of Stackers are honest, then sBTC remains safe.**  Every BTC
+peg-out is paired with an equivalent, legitimate request from a wrapped BTC
+holder, _even if every miner is malicious_.  This is achieved by ensuring that
+all Stacking and peg-maintenance operations materialize in all Stacks forks.
+
+* **If a Byzantine fault-tolerant (BFT) majority of miners are honest, then sBTC
+remains live**.  All peg-out requests are eventually fulfilled, _even if every
+Stacker is malicious_.  This is because Stackers do not have a say in block
+production, and Stackers that do not fulfill peg-outs lose their PoX-driven BTC
+income from miners.
+
+* **Peg-outs of arbitrary amounts of BTC are fulfilled in
+a fixed amount of time on the happy path** if both Stackers and miners operate with a BFT honest
+majority.  If they do not, then peg-outs of arbitrary amounts of BTC 
+are fulfilled eventually by _redirecting_ Stackers' PoX payouts to fund peg-out
+requests.
+
+To support this, sBTC offers two modes of operation:
+the Normal mode, and the Recovery mode.  In **Normal mode**, the sBTC asset is
+backed 1:1 by BTC sent to a wallet controlled by a large fraction of Stackers,
+as measured by the fraction of locked STX they represent.  Each time BTC is
+sent to this wallet (a peg-in operation), an equal number of sBTC are
+automatically minted to an address of the sender's choosing.  Stackers respond
+to peg-out requests by sending BTC to the requester's Bitcoin address, thereby
+maintaining the peg.  The act of responding to peg-out requests automatically
+destroys the requester's equivalent amount of sBTC on the Stacks chain.
+
+If the Normal mode encounters a liveness failure for any reason (including loss
+of BTC), the system transitions to a Recovery mode until enough Stackers come
+back online.  In **Recovery mode**, a fraction of the PoX payouts are
+redirected to peg-out requests such that eventually, all outstanding requests
+will be fulfilled, even if the Stackers _never_ come back online.  While
+considerably slower than Normal mode, the design of Recovery mode ensures all
+sBTC can be redeemed so long as the Stacks blockchain and PoX are online.
```

Comment:
[nit] "PoX are online" sounds weird (PoX not a separate system or service). Maybe "so long as the Stacks blockchain is operational and PoX is active" is clearer?

Reply:
PoX is a subsystem within the blockchain that can be disabled while the blockchain is online.